components/iostreams/tests/regressions/lost_output_2236.cpp

```diff
@@ -6,7 +6,7 @@
 
 #include <hpx/hpx.hpp>
 #include <hpx/hpx_init.hpp>
-#include <hpx/include/iostreams.hpp>
+#include <hpx/iostream.hpp>
```

Comment:
This we don't have yet. `hpx/iostream.hpp` should most likely be a "local" version of our current iostream support, and this particular include would have to be `hpx/distributed/iostream.hpp`. If you'd like to add the latter you can do so here.

Reply:
Ah, please ignore the part about adding the file. I see you've already done that. But do change it to `hpx/distributed/iostream.hpp`.

Reply:
Ok perfect will do :)